app/Http/Controllers/PackageController.php

```diff
@@ -26,4 +27,12 @@ public function show($namespace, $name)
             ->with('package', PackageDetailResource::from($package))
             ->with('screenshots', $package->screenshots);
     }
+
+    public function showId(Package $package)
+    {
+        return redirect()->route('packages.show', [
```

Comment:
Sorry to make you modify again, but I just hunted down my original code:

```php
        return redirect()->route('packages.show', [
            'namespace' => $package->composer_vendor,
            'name' => $package->composer_package,
        ]);
```

Reply:
ah, that `composer_package` and `composer_vendor` is handy. Fixed!